### ISSUE TITLE
v2版本的论文什么时候提交到arxiv上？

### ISSUE THREAD

**Issue by bojone** [jquesnelle/yarn#22]
如题。对于强迫症来说，目前的版本看着有点难受。

**Comment by Cebtenzzre** [comment]
[Translation: When will the v2 version of the paper be submitted to arxiv?]

**Comment by bloc97** [comment]
Yes, we are planning on adding more data and graphs to the paper before trying peer review, we will try our best to publish v2 as soon as possible, but keep in mind that as every person on the team is currently working on this out of their free time, it might take slightly more time. Since we don't want to spam arXiv with too many versions, I can update the corrected version as a PDF on this repo...
